tap_shopify/streams/transactions.py

```diff
@@ -23,29 +23,34 @@
 # their values are not equal
 def canonicalize(transaction_dict, field_name):
     field_name_upper = field_name.capitalize()
-    value_lower = transaction_dict.get('receipt', {}).get(field_name)
-    value_upper = transaction_dict.get('receipt', {}).get(field_name_upper)
-    if value_lower and value_upper:
-        if value_lower == value_upper:
-            LOGGER.info((
-                "Transaction (id=%d) contains a receipt "
-                "that has `%s` and `%s` keys with the same "
-                "value. Removing the `%s` key."),
+    # Not all Shopify transactions have receipts. Facebook has been shown
+    # to push a null receipt through the transaction
+    receipt = transaction_dict.get('receipt', {})
+    if receipt:
```

Comment:
This is really the only change in this PR. All the code below this is simply whitespace change from indenting the code under this `if receipt` check.